index.test-d.ts

```diff
@@ -139,55 +151,55 @@ execa('unicorns').kill('SIGKILL', {forceKillAfterTimeout: false});
+expectType<ExecaSyncReturnValue>(execaSync('unicorns'));
+expectType<ExecaSyncReturnValue>(
+	execaSync('unicorns', {encoding: 'utf8'}),
 );
 expectType<ExecaSyncReturnValue<Buffer>>(
-	execa.sync('unicorns', {encoding: null})
+	execaSync('unicorns', {encoding: null}),
 );
-expectType<ExecaSyncReturnValue<string>>(
-	execa.sync('unicorns', ['foo'], {encoding: 'utf8'})
+expectType<ExecaSyncReturnValue>(
+	execaSync('unicorns', ['foo'], {encoding: 'utf8'}),
 );
 expectType<ExecaSyncReturnValue<Buffer>>(
-	execa.sync('unicorns', ['foo'], {encoding: null})
+	execaSync('unicorns', ['foo'], {encoding: null}),
 );
 
-expectType<ExecaChildProcess<string>>(execa.command('unicorns'));
-expectType<ExecaReturnValue<string>>(await execa.command('unicorns'));
-expectType<ExecaReturnValue<string>>(await execa.command('unicorns', {encoding: 'utf8'}));
-expectType<ExecaReturnValue<Buffer>>(await execa.command('unicorns', {encoding: null}));
-expectType<ExecaReturnValue<string>>(await execa.command('unicorns foo', {encoding: 'utf8'}));
-expectType<ExecaReturnValue<Buffer>>(await execa.command('unicorns foo', {encoding: null}));
+expectType<ExecaChildProcess>(command('unicorns'));
+expectType<ExecaReturnValue>(await command('unicorns'));
+expectType<ExecaReturnValue>(await command('unicorns', {encoding: 'utf8'}));
+expectType<ExecaReturnValue<Buffer>>(await command('unicorns', {encoding: null}));
+expectType<ExecaReturnValue>(await command('unicorns foo', {encoding: 'utf8'}));
+expectType<ExecaReturnValue<Buffer>>(await command('unicorns foo', {encoding: null}));
 
-expectType<ExecaSyncReturnValue<string>>(execa.commandSync('unicorns'));
-expectType<ExecaSyncReturnValue<string>>(execa.commandSync('unicorns', {encoding: 'utf8'}));
-expectType<ExecaSyncReturnValue<Buffer>>(execa.commandSync('unicorns', {encoding: null}));
-expectType<ExecaSyncReturnValue<string>>(execa.commandSync('unicorns foo', {encoding: 'utf8'}));
-expectType<ExecaSyncReturnValue<Buffer>>(execa.commandSync('unicorns foo', {encoding: null}));
+expectType<ExecaSyncReturnValue>(commandSync('unicorns'));
+expectType<ExecaSyncReturnValue>(commandSync('unicorns', {encoding: 'utf8'}));
+expectType<ExecaSyncReturnValue<Buffer>>(commandSync('unicorns', {encoding: null}));
+expectType<ExecaSyncReturnValue>(commandSync('unicorns foo', {encoding: 'utf8'}));
+expectType<ExecaSyncReturnValue<Buffer>>(commandSync('unicorns foo', {encoding: null}));
 
-expectType<ExecaChildProcess<string>>(execa.node('unicorns'));
-expectType<ExecaReturnValue<string>>(await execa.node('unicorns'));
-expectType<ExecaReturnValue<string>>(
-	await execa.node('unicorns', {encoding: 'utf8'})
+expectType<ExecaChildProcess>(node('unicorns'));
+expectType<ExecaReturnValue>(await node('unicorns'));
+expectType<ExecaReturnValue>(
+	await node('unicorns', {encoding: 'utf8'}),
 );
-expectType<ExecaReturnValue<Buffer>>(await execa.node('unicorns', {encoding: null}));
-expectType<ExecaReturnValue<string>>(
```

Comment:
Nice catch with the unnecessary `<string>`. :+1:

Reply:
Haha wasn't me. It was all `xo --fix` :)